### PR TITLE
[AB#56] Unused files removed.

### PR DESCRIPTION
**Pull request by lesair**
It looks safe to the rid of these. What do you guys think?

### CODE REVIEW

**Comment by lesair** [comment]
@limonxz and @janus78: what do you think?

**Comment by limonxz** [comment]
Oh, yeah! is ok!

**Review by lesair — COMMENTED** [review]
Looks good!

File: src/withRoot.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    error: red
+  },
+  typography: {
+    useNextVariants: true
+  }
+});
+
+function withRoot(Component) {
+  function WithRoot(props) {
+    // MuiThemeProvider makes the theme available down the React tree
```
Comment (limonxz) [inline src/withRoot.js:24]: I dont understand why are this function nested, could you explain me?

**Review by limonxz — APPROVED** [review]
Only I need the explanation from my question

**Pull request merged**